tools/bake.py

```diff
@@ -1029,6 +1029,22 @@ def Scale2D( v, s, p ):
                 if not overlap_aware:
                     bpy.ops.uv.pack_islands(rotate=True, margin=margin)
                 try:  # UVP doesn't respect margins when called like this, find out why
+                    context.scene.uvpm3_props.normalize_islands = False
+                    context.scene.uvpm3_props.lock_overlapping_enable = True
+                    context.scene.uvpm3_props.lock_overlapping_mode = ('1' if
+                                                                      layer == 'CATS UV Super' else
+                                                                      '2')
+                    context.scene.uvpm3_props.pack_to_others = False
+                    context.scene.uvpm3_props.margin = margin
+                    context.scene.uvpm3_props.simi_threshold = 3
+                    context.scene.uvpm3_props.precision = 1000
+                    # Give UVP a static number of iterations to do TODO: make this configurable?
+                    for _ in range(1, 3):
```

Comment:
I'm also not sure this is actually doing anything... 
There's now a 
` bpy.context.scene.uvpm3_props.heuristic_enable = True`

and a 
`bpy.context.scene.uvpm3_props.heuristic_enable.max_wait_time`
as well as a
`bpy.context.scene.uvpm3_props.heuristic_enable.search_time`

I'm not sure if subsequent iterations used to improve results in uvpm2. I'm also not sure of decent default heuristic search settings it might just be worth removing this for loop for now and also leaving heuristics defaulted to off?